peoplepage.html

```diff
@@ -59,15 +85,15 @@
                         <li class="nav-item dropdown px-lg-3">
                             <a class="nav-link active dropdown-toggle" href="" role="button" data-bs-toggle="dropdown">Categories</a>
                               <ul class="dropdown-menu" id="categoryItems">
```

Comment:
missing some javascript to add the category list items

Reply:
in progress...
stuck to this script.

Reply:
`        // Get Genres
        const fetchGenreData = () => {
            fetch('https://api.themoviedb.org/3/genre/movie/list', options)
                .then(response => response.json())
                .then(response => {
                    response = response.genres;
                    response.forEach((result) => {
                        //dNavbar Category Display
                        document.querySelector("#categoryItems").innerHTML += `
                            <li>
                                <a class="dropdown-item" href="./search.html?list=${result.id}">${result.name}</a>
                            </li>
                        `;
                    });
                })
        }

        fetchGenreData();
`

Reply:
`
        const fetchGenreData = () => {
            fetch('https://api.themoviedb.org/3/genre/movie/list', options)
                .then(response => response.json())
                .then(response => {
                    response = response.genres;
                    response.forEach((result) => {
                        //dNavbar Category Display
                        document.querySelector("#categoryItems").innerHTML += `
                            <li>
                                <a class="dropdown-item" href="./search.html?list=${result.id}">${result.name}</a>
                            </li>
                        `;
                    });
                })
        }
        fetchGenreData();
`

Reply:
Done. Kindly check if it works.